build.zig

```diff
@@ -930,6 +1089,267 @@ test hex64 {
     try std.testing.expectEqualStrings("[00efcdab78563412]", s);
 }
 
+// ------------------------------------------
+// SPIR-V include generation logic
+// ------------------------------------------
+
+fn ensurePython(allocator: std.mem.Allocator) void
+{
+    if (!ensureCommandExists(allocator, "python3", "--version"))
```

Comment:
we can't have a dependency on python here

This is a bit unfortunate; I am guessing we would need a fork of https://github.com/KhronosGroup/SPIRV-Headers packaged for build.zig with the headers pregenerated

Reply:
Got it. I'll make a repo with the headers pregenerated.